src/BridgeHelper/XDBCBridgeHelper.h

```diff
@@ -96,9 +96,14 @@ class XDBCBridgeHelper : public IXDBCBridgeHelper
 
     bool bridgeHandShake() override
     {
+        const auto & settings = getContext()->getSettingsRef();
+        const auto & cfg = getContext()->getConfigRef();
+        Poco::Timespan http_keep_alive_timeout{cfg.getUInt("keep_alive_timeout", 10), 0};
+        auto timeouts = ConnectionTimeouts::getHTTPTimeouts(settings, http_keep_alive_timeout);
```

Comment:
It feels really ugly to repeat this code over and over... Is there still a way to do it in some common place? Or at least hide the constant?

Reply:
Agree, will make this less verbose.